### PR TITLE
add clipboard handler

### PR DESCRIPTION
**Pull request by themoritz**
Once https://github.com/purescript-contrib/purescript-dom/pull/82 and https://github.com/slamdata/purescript-dom-indexed/pull/4 are ready.

### CODE REVIEW

File: src/Halogen/HTML/Events.purs [hunk 1]
```diff
@@ -107,6 +105,15 @@ onSubmit = handler (EventType "submit")
 onTransitionEnd :: forall r i. (Event -> Maybe i) -> IProp (onTransitionEnd :: Event | r) i
 onTransitionEnd = handler (EventType "transitionend")
 
+onCopy :: forall r i. (ClipboardEvent-> Maybe i) -> IProp (onCopy :: ClipboardEvent | r) i
```
Comment (vendethiel) [inline src/Halogen/HTML/Events.purs]: minor: missing a space before `-> Maybe i` (three times)

File: src/Halogen/HTML/Events.purs [hunk 2]
```diff
@@ -45,24 +48,19 @@ module Halogen.HTML.Events
...
-
-import Halogen.Query (Action, action)
-import Halogen.Query.InputF (InputF(..))
 import Halogen.HTML.Core (Prop)
-import Halogen.HTML.Core as Core
 import Halogen.HTML.Properties (IProp)
-
+import Halogen.Query (Action, action)
+import Halogen.Query.InputF (InputF(..))
 import Unsafe.Coerce (unsafeCoerce)
```
Comment (garyb) [inline src/Halogen/HTML/Events.purs]: Minor nitpick, can you sort the imports alphabetically please?

**Comment by garyb** [comment]
Thanks! Will merge & release once Travis completes.

File: src/Halogen/HTML/Events.purs (hunk 2 above)
Comment (themoritz) [inline src/Halogen/HTML/Events.purs]: Sure!

**Pull request merged**